### DISCUSSION TITLE
Help with relations

### DISCUSSION THREAD

**Discussion by ChrisF999** [prisma/prisma#6608] [Q&A] [answered]
The error I am getting is:
```
 error: Error parsing attribute "@relation": The type of the field `team_twitch_alert_id` in the model `UserInTwitchTeam` is not matching the type of the referenced field `team_twitch_alert_id` in model `TeamInTwitchAlert`.
  -->  schema.prisma:140
   | 
139 |   message_id           Int?
140 |   TeamInTwitchAlert    TeamInTwitchAlert @relation(fields: [team_twitch_alert_id], references: [team_twitch_alert_id])
141 | 
   | 
```
I haven't a clue in whats causing it theses shemas were made by  npx prisma introspect.
Any help would be apricated. 

```
model UserInTwitchTeam {
  team_twitch_alert_id String
  twitch_username      String
  message_id           Int?
  TeamInTwitchAlert    TeamInTwitchAlert @relation(fields: [team_twitch_alert_id], references: [team_twitch_alert_id])

  @@id([team_twitch_alert_id, twitch_username])
}

```
```
model TeamInTwitchAlert {
  team_twitch_alert_id Int                @id @default(autoincrement())
  channel_id           Int
  twitch_team_name     String
  custom_message       String?
  TwitchAlerts         TwitchAlerts       @relation(fields: [channel_id], references: [channel_id])
  UserInTwitchTeam     UserInTwitchTeam[]
```

**Comment by rikoriswandha** [comment, marked as answer]
The message is quite self explanatory. Your `team_twitch_alert_id` inside `UserInTwitchTeam` is String while `team_twitch_alert_id` inside `TeamInTwitchAlert` is Int type

**Reply by ChrisF999** [reply]
Oh i see now, don't really see how i missed that.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
